frontend/src/views/Main.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { Navigation } from "../layout/Navigation";
+
+export const Main = () => {
+    return (
+        <>
+            <h1 className="title">Main</h1>
```

Comment:
You call this "Home" in the nav menu, which honestly makes more sense. Can you update the file name and header here to also be "Home"?

Reply:
Done